JEasyCryptoClient/src/CryptoClient.java

```diff
@@ -39,7 +38,9 @@ public void run() {
 				console.printf("Quitting CryptoClient!\n");
 				return;
 			}
-			socket = new DatagramSocket(clientPort);
+			// Construct socket on any available port
+			socket = new DatagramSocket(0);
+			console.printf("Allocated client port is %d.\n", socket.getLocalPort());
```

Comment:
It would be good for the user, if the actual port would be more easily visible in the printed text. Maybe we can use a colon?

Reply:
Yes, updated as requested.